src/components/Chart/Chart.tsx

```diff
@@ -383,6 +389,7 @@ function Chart({
   infoTooltipMessage: string;
   expandedGraphStrokeColor: string;
   isPOL: boolean;
+  margin?: any;
```

Comment:
Please don't use the 'any' type. 
You should use 
```suggestion
  margin?: CategoricalChartProps["margin"]
```